src/lightning_lite/lite.py

```diff
@@ -345,6 +345,24 @@ def all_gather(
     def broadcast(self, obj: TBroadcast, src: int = 0) -> TBroadcast:
         return self._strategy.broadcast(obj, src=src)
 
+    # TODO: Find intuitive name
+    @contextmanager
+    def create_sharded_model(self) -> Generator:
```

Comment:
Many names are floating around, all of them are kinda meh.

```py
with sharding():

with model_parallel():

with create_sharded_model():

with sharded_model()
```

Looking for suggestions.

Reply:
I like 4)